src/i18n/zh_TW.jsonc

```diff
@@ -43,8 +43,13 @@
   "term.albums": "專輯",
   "term.artists": "藝人",
   "term.podcasts": "Podcasts",
-  "term.playlists": "播放列表",
-  "term.playlist": "播放列表",
+  "term.playlists": "播放清單",
```

Comment:
See comment

Reply:
![Snipaste_2022-02-10_01-39-58](https://user-images.githubusercontent.com/17287792/153258605-b0af84e8-55e7-4509-9300-6481a5cb29d5.png)
Got it. I use iTunes translation here, probably deprecated.